composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
+        "ext-hash": "*",
```

Comment:
see https://www.php.net/manual/en/hash.installation.php

as of PHP 5.1.2 always enabled when not disabled manually

as of PHP 7.4 always available